tests/unit/job/test_query_pandas.py

```diff
@@ -119,25 +121,43 @@ def test_to_dataframe_bqstorage_preserve_order(query, table_read_options_kwarg):
         },
         "totalRows": "4",
     }
-    connection = make_connection(get_query_results_resource, job_resource)
+    stream_id = "projects/1/locations/2/sessions/3/streams/4"
+    name_array = pyarrow.array(
+        ["Matthew", "Mark", "Luke", "John"], type=pyarrow.string()
```

Comment:
```suggestion
        ["John", "Paul", "George", "Ringo"], type=pyarrow.string()
```

want to remove religious references to play it safe around code of conduct